packages/6-todo-react/src/components/todo-app/TodoApp.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { FC, ReactElement, useContext, useEffect } from 'react';
+import { jss } from '../../styles/jss';
+import { colors, commonStyles } from '../../styles/commonStyles';
+import { Context, GlobalContext } from '../../context/Store';
+import { ITodoDTO } from 'fed-todo-journey_todo-common';
+import { TodoItem } from './todo-item/TodoItem';
+import { TodoInput } from './todo-input/TodoInput';
+import { todosService } from '../../services/TodosService';
+import { TodosActionTypes } from '../../context/TodosActions';
+
+export const TodoApp: FC<{}> = (): ReactElement => {
+  const { state, dispatch } = useContext<GlobalContext>(Context);
```

Comment:
why did u choose redux (i.e, reducers, actions, dispatch) for such a simple app? seems like an overkill.
1. u add the complexity of knowning redux to your fellow team members (that might help u in the app in the future)
2.in general redux is pretty deprecated. context and providers can solve the state management problem for most apps.